Tests/Integration/PauseWithPredisTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+namespace Resque\Plugins\Tests\Integration;
+
+use Resque;
+use Predis\Client;
+
+/**
+ * Pause tests.
+ *
+ * @package     PHP Resque Pause
+ * @author      Wedy Chainy <wedy.chainy@bigcommerce.com>
+ * @license     http://www.opensource.org/licenses/mit-license.php
+ */
+class PauseWithPredisTest extends PauseTestBase
+{
+    public static function setUpBeforeClass()
+    {
+        $testMisc = realpath(__DIR__ . '/misc/');
+        $redisConf = "$testMisc/redis.conf";
+
+        // Attempt to start our own redis instance for tesitng.
+        exec('which redis-server', $output, $returnVar);
+        if ($returnVar != 0) {
+            echo "Cannot find redis-server in path. Please make sure redis is installed.\n";
+            exit(1);
+        }
+
+        exec("cd $testMisc; redis-server $redisConf", $output, $returnVar);
+        usleep(500000);
```

Comment:
Why the sleep? Does it wait half a second to start the server before it's ready? It should probably be in an else block so that if it fails straight away it doesn't wait. Not that half a second is a long time. It should also have a comment as to why you're waiting.